pypeit/spectrographs/keck_lris.py

```diff
@@ -1085,6 +1102,35 @@ def get_detector_par(self, det, hdu=None):
         # Return
         return detector
 
+
+    def parse_dither_pattern(self, file_list, ext=None):
```

Comment:
Does this method do anything?  If not, why is it here?  Seems like it might go best in the `spectrograph` class.

Reply:
Done.